app/hyperterm.js

```diff
@@ -263,6 +265,32 @@ export default class HyperTerm extends Component {
     }
   }
 
+  increaseFontSize () {
+    const uid = this.state.sessions[this.state.active];
+    const term = this.refs[`term-${uid}`];
+    if (term) {
+      try {
+        const size = term.term.prefs_.get('font-size');
+        term.term.prefs_.set('font-size', size + 1);
+      } catch (e) {
+        alert(e);
+      }
+    }
+  }
+
+  decreaseFontSize () {
+    const uid = this.state.sessions[this.state.active];
+    const term = this.refs[`term-${uid}`];
+    if (term) {
+      try {
+        const size = term.term.prefs_.get('font-size');
+        term.term.prefs_.set('font-size', size - 1);
+      } catch (e) {
+        alert(e);
+      }
+    }
+  }
```

Comment:
This functions are almost identical. I would DRY them up into one that accepted an argument of whether to increment or decrement.